lib/Infer/AliveDriver.cpp

```diff
@@ -300,6 +301,26 @@ bool souper::AliveDriver::translateRoot(const souper::Inst *I, const Inst *PC,
   return true;
 }
 
+// Dummy because it doesn't actually build expressions.
+// It exists for the purpose of reusing parts of the abstract ExprBuilder here.
```

Comment:
ugh... are these expected to be called? if not, make them throw a fatal error
also please put a FIXME to remove them when possible